wordle-app/pages/index.vue

```diff
@@ -0,0 +1,87 @@
+<template>
+  <v-container>
+
```

Comment:
May want to add a background image as per Assignment4.md instructions. Though I'm not sure if its a hard requirement or just a recommendation.

Reply:
They have one in the `app.vue` but its a video. I think thats okay but maybe want to ask Meg or Grant about it.

Reply:
I removed a video and added an animated gradient. Grant said that it was okay.